source/concepts/account.rst

```diff
@@ -37,6 +37,19 @@ Address
 Each account has a unique :ref:`address <address>` derived from the public key.
 Normally, the address is shared instead of the public key because it is shorter and gathers information about the network.
 
+Hierarchical Deterministic Wallets
+===========
+
+Symbol supports a multi-account hierarchy enabling the generation of multiple public keys without revealing the private key.
+
+The deterministic keys from which the accounts will be derived from are serialized into human-readable words in Mnemonic codes.
```

Comment:
I would explain what's a mnemonic code the number of words that compose the passphrase and how it's related to the parent private key (or seed).